spring-boot-admin-server-ui/src/main/frontend/views/applications/index.vue

```diff
@@ -217,32 +247,49 @@
       }
 
       return applications
-        .map(a => a.filterInstances(i => instanceMatchesFilter(this.filter.toLowerCase(), i)))
-        .filter(a => a.instances.length > 0);
-    }
+        .map((a) =>
+          a.filterInstances((i) =>
+            instanceMatchesFilter(this.filter.toLowerCase(), i)
+          )
+        )
+        .filter((a) => a.instances.length > 0);
+    },
   },
   computed: {
     statusGroups() {
       const filteredApplications = this.filterInstances(this.applications);
-      const applicationsByStatus = groupBy(filteredApplications, application => application.status);
-      const list = transform(applicationsByStatus, (result, applications, status) => {
-        const statusKey = status.replace(/[^\w]/gi, '').toLowerCase();
-        result.push({statusKey, status: status, applications: sortBy(applications, [application => application.name])})
-      }, []);
-      return sortBy(list, [item => item.status]);
-    }
+      const applicationsByStatus = groupBy(
+        filteredApplications,
+        (application) => application.status
+      );
+      const list = transform(
+        applicationsByStatus,
+        (result, applications, status) => {
+          const statusKey = status.replace(/[^\w]/gi, "").toLowerCase();
+          result.push({
+            statusKey,
+            status: status,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/ulischulte/spring-boot-admin/pullRequest?prid=8294305)
```suggestion
            status,
```